test/download-test.js

```diff
@@ -589,6 +590,18 @@ describe('Download video', () => {
     });
   });
 
+  describe('With IPv6 Block', () => {
+    it('Sends request with IPv6 address', done => {
+      const stream = ytdl.downloadFromInfo(testInfo, { IPv6Block: '2001:2::/48' });
+      stream.on('info', (info, format) => {
+        nock.url(format.url).reply(function checkAddr() {
+          assert.ok(net.isIPv6(this.req.options.localAddress));
+          done();
+        });
+      });
+    });
+  });
+
```

Comment:
```suggestion

  describe('Without IPv6 Block', () => {
    it('Sends request with (default) IPv4 address', done => {
      const stream = ytdl.downloadFromInfo(testInfo);
      stream.on('info', (info, format) => {
        nock.url(format.url).reply(function checkAddr() {
          assert.ok(net.isIPv4(this.req.options.localAddress));
          done();
        });
      });
    });
  });
```

Reply:
disclaimer - did not try this locally

Reply:
ok, sorry for a bit of inactivity from my side but ill do testing for all those changes today/this week(to make sure everything works as expected), fix conflicts and then we good i guess

Reply:
added in recent commit